.gitignore

```diff
@@ -9,4 +9,16 @@ certs/*.srl
 cluster-config/cluster-cfg.yaml
 .DS*
 idsvr-config/templates/deployed-idsvr-helm.yaml
-templates
\ No newline at end of file
+templates
+.terraform
+.terraform.lock.hcl
+terraform.tfstate
+terraform-config/.terraform.tfstate.lock.info
+terraform-config/terraform.tfstate.backup
+terraform-config/1.4-vpc-creation.auto.tfvars
+terraform-config/2.4-eks-cluster-deployment.auto.tfvars
+.gitignore
+.gitignore
```

Comment:
Why remove `.gitignore` from version control?
I assume git should ignore any `*.tfvars` file as well as `tfstate` files. 

Suggest to use the following entries in `.gitignore`:
```
# Local folder
.terraform/

# Dependency file
.terraform.lock.hcl

# Compiled files
*.tfstate
*.tfstate.backup
*.tfstate.lock.info

# Variable files
*.tfvars
```

Reply:
Fixed.